Jenkinsfile

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env groovy
 
-/* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
-buildPlugin()
+// Use recommended configuration, run all tests to completion (don't fail fast)
+buildPlugin(configurations: buildPlugin.recommendedConfigurations(),
```

Comment:
revert please this isn't what you want, recommendedConfigurations is not recommended anymore